core/lib/multivm/src/tracers/call_tracer/vm_latest/mod.rs

```diff
@@ -185,9 +183,11 @@ impl CallTracer {
             return;
         };
 
+        self.call_depth_on_prefix.pop();
```

Comment:
seems you have to pop only for far calls

Reply:
Generally yes, but since we doing pop for stack and after that it can push something to stack again, where it will recalculate new value for prefix, it should work okay

Reply:
So it will anyway be pushed again and length of call stack will be the same with prefix vec length all the time

Reply:
Let's write a test :)